clients/admin-ui/src/features/data-discovery-and-detection/hooks/useDiscoveryResultColumns.tsx

```diff
@@ -17,6 +17,9 @@ import DiscoveryItemActionsCell from "../tables/cells/DiscoveryItemActionsCell";
 import EditCategoryCell from "../tables/cells/EditCategoryCell";
 import ResultStatusCell from "../tables/cells/ResultStatusCell";
 
+const NAME_COLUMN_SIZE = 300;
+const ACTION_COLUMN_SIZE = 220;
```

Comment:
This one is slightly wider since it potentially needs to accommodate a kebab menu also.

Reply:
```suggestion
const ACTION_COLUMN_SIZE = 235;
```
Good catch! I did found that if I intentional shink the column down to it's minimum width it will still generate a bit of scrolling. Bringing it up a to 235 removes that scroll.

<img width="377" alt="Captura de pantalla 2025-02-26 a la(s) 2 57 04 p  m" src="https://github.com/user-attachments/assets/a434ae6d-ec5f-43af-b8ee-811e924c86f0" />